config/crd/bases/ssp.kubevirt.io_ssps.yaml

```diff
@@ -22,21 +22,27 @@ spec:
         description: SSP is the Schema for the ssps API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
All changes in this file are just adding newlines to `description` fields. I'm not sure why, but for some developers the `operator-sdk` adds newlines and for some it does not. It probably depends on the environment. We will need to investigate later.

Anyway, this change is not part of this PR, so you don't need to commit this file.